portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -133,14 +166,23 @@ private String getParameter(HttpServletRequest request, String name, String defa
    */
   class Comment {
     // The fields that hold the relevant comment data.
+    private long id;
```

Comment:
I think the id needs it's own comment, since it's not always set, it would be good to explain what a default value means.

Reply:
Ok, I added another comment explaining what the ID is, and what a default value means.